src/providers/fakerest/internal/transformFilter.ts

```diff
@@ -0,0 +1,54 @@
+import { transformContainsFilter } from './transformContainsFilter';
+import { transformInFilter } from './transformInFilter';
+import { transformOrFilter } from './transformOrFilter';
+
+export function transformFilter(filter: Record<string, any>) {
+    const transformedFilters: Record<string, any> = {};
+    for (const [key, value] of Object.entries(filter)) {
+        if (
+            key.endsWith('@eq') ||
+            key.endsWith('@neq') ||
+            key.endsWith('@lt') ||
+            key.endsWith('@lte') ||
+            key.endsWith('@gt') ||
+            key.endsWith('@gte')
+        ) {
+            const lastIndexOfAt = key.lastIndexOf('@');
+            transformedFilters[
+                `${key.substring(0, lastIndexOfAt)}_${key.substring(lastIndexOfAt + 1)}`
+            ] = value;
+            continue;
+        }
+
+        if (key.endsWith('@is')) {
+            transformedFilters[`${key.slice(0, -3)}_eq`] = value;
+            continue;
+        }
```

Comment:
**question**: have you checked how this fares with nullish values? I recall having issues with fakerest not being able to filter null values correctly...

Reply:
It seemed to work during my tests